JapaneseFilter/sections/specific.txt

```diff
@@ -2201,6 +2201,8 @@ game-news.tokyo###bottomNbox
 game-news.tokyo##.wide_adbox
 tabizine.jp##div[id$="_parent"]
 anichoice.com,yosakoimatsuri.com##.widgetPost-bottom
+sirabee.com##.articleBottomBanner
+sirabee.com##div[style="min-height:250px;min-width:300px"]
```

Comment:
It doesn't match on my end. If I understand correctly, you can add to `idou.me##.side_banner` for the same placeholder.

Reply:
Also add the domain to `toyokeizai.net##.related` for another leftover.

Reply:
I see.